package.json

```diff
@@ -97,7 +97,7 @@
     "webpack-dev-server": "2.11.2"
   },
   "dependencies": {
-    "@tendermint/ui": "0.2.13",
+    "@tendermint/ui": "https://github.com/tendermint/ui.git",
```

Comment:
Should we go with this? Or should we wait with this PR until a release?

Reply:
@nylira could we update the npmjs package?

Reply:
Just published 0.2.15!